yetanotherco/aligned_layer
# feat (verifier): add risc_zero verifier ffi

# Description
- This PR adds the Risc0 verifier FFI.
- To test, run:
  - `make generate_risc_zero_fibonacci_proof`. You will see `error: invalid inline table` but this is a problem of the risc0 library, the program will compile anyways.
  - Then, run `make test_risc_zero_rust_ffi`. To run the rust tests.
  - Finally, run `make test_risc_zero_go_bindings_macos` or `make test_risc_zero_go_bindings_linux` depending on your OS to test the go bindings.
